### DISCUSSION TITLE
Open applications in different workspaces on boot

### DISCUSSION THREAD

**Discussion by IgnisDa** [hyprwm/Hyprland#1421] [Q&A]
I want that different applications be opened up in different workspaces on start-up. I am aware of `exec-once` but how can I adapt it to my usecase?

**Comment by vaxerski** [comment]
`exec-once=[workspace 2]kitty`

**Reply by IgnisDa** [reply]
I also want it so that `workspace 2` get assigned to a specific monitor.

**Reply by vaxerski** [reply]
`bindws=mon,workspace` for permanent, `exec-once=[monitor X;workspace 2]kitty` for once

**Reply by IgnisDa** [reply]
> exec-once=[monitor X;workspace 2]kitty

Is this documented somewhere?

**Reply by vaxerski** [reply]
of course: https://wiki.hyprland.org/Configuring/Dispatchers/#executing-with-rules
